resources/js/markdown/renderers.tsx

```diff
@@ -54,6 +54,7 @@ export function timestampDecorator(reactNode: React.ReactNode): React.ReactNode
 
         // decorate the timestamp as a link
         const [,,, m, s, ms, range] = match;
+        // TODO: look at noUncheckedIndexedAccess
```

Comment:
typescript assumes destructing an array this way doesn't have `undefined` values even though something like `const [a,b] = [1] makes `b` `undefined` but not typed as nullable.

Reply:
I feel whichever option is used the regexp match handling will be questionable unless the compiler actually looks at the regexp and figures out which ones could be null

Reply:
Well, it looks like `noUncheckedIndexedAccess` just creates the inverse problem where it assumes everything can be `undefined` after a bounds check 🤣 ; probably better to just suppress the error and add a note since the more common case is when we know which value will be undefined